core/domain/draft_upgrade_services.py

```diff
@@ -88,6 +88,70 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v30_dict_to_v31_dict(cls, draft_change_list):
+        """Converts draft change list from state version 30 to 31. State
+        Version 31 adds the duration float for the Voiceover section of state.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        for i, change in enumerate(draft_change_list):
+            if (change.cmd == exp_domain.CMD_EDIT_STATE_PROPERTY and
+                    change.property_name ==
+                    exp_domain.STATE_PROPERTY_RECORDED_VOICEOVERS):
+                # Get the language code to access the language code correctly.
+                new_change = change.new_value['voiceovers_mapping']
+                language_code = new_change['content'].keys()
+                # Initialize the value to migrate draft state to v31.
+                new_change['content'][language_code[0]]['duration'] = 0.0
```

Comment:
Why are we using language_code[0] (what is special about the first language code in the list)?

Reply:
Problem is that you would not know what language the voiceover is in. I can't assume that every voiceover will be in english. Unless there is a better way access that is language agnostic.

Reply:
Found a way going to try this.

Reply:
Sorry, I'm still not sure I understand what's going on. Could you explain what you're trying to do here, exactly? Does it depend on which specific language is being used?

Reply:
I have provided an image showing the data dict to be accessed. 
![Screenshot-from-2020-01-15-20-32-01--cro ed](https://user-images.githubusercontent.com/9009585/73421137-0872ba00-42e2-11ea-84b1-052a0c82401f.png)
As you can see in the red box you can see the 'en' key which signifies that this is an English voiceover. Other languages like arabic have 'ar' as key. To clarify I needed a way to use the key in an agnostic matter. I applied this to anything that had a dynamic key for the dict.

Reply:
Ah, I see. So, just to check ... what do you plan to do if there is more than one key-value pair in the dict?

Reply:
(I would have expected something like "iterate over all the key-value pairs and do XXX for each of them".)

Reply:
I can adjust to do multiple pair keys or modify the for loop to be like
`for key,value in change.new_value.items():` . Then I don't have to do keys() stuff.

Reply:
The latter seems good to me, assuming that that does the correct thing. My original issue was with selecting just the first element out of what might be a list of multiple elements -- I was trying to understand whether that was the correct thing to do or not.